std/math.d

```diff
@@ -7525,10 +7525,13 @@ if (isFloatingPoint!(F) && isIntegral!(G))
 typeof(Unqual!(F).init * Unqual!(G).init) pow(F, G)(F x, G n) @nogc @trusted pure nothrow
 if (isIntegral!(F) && isIntegral!(G))
 {
-    if (n<0) return x/0; // Only support positive powers
```

Comment:
Why does this runtime check exist instead of requiring the exponent to be unsigned?

Reply:
This is a good reason to: https://forum.dlang.org/post/qobojhyjmxzvxorlhhnj@forum.dlang.org